AppInspector/rules/default/os/system_registry.json

```diff
@@ -42,6 +42,43 @@
       }
     ]
   },
+  {
+    "name": "OS Operation: Windows Registry (Write)",
+    "id": "AI038110",
+    "description": "OS Operation: Windows Registry (Write)",
+    "tags": [
+      "OS.SystemRegistry.Write"
+    ],
+    "applies_to": [
+      "csharp"
+    ],
+    "severity": "moderate",
+    "patterns": [
+      {
+        "pattern": "\\.create_subkey\\(|\\.set_value\\(",
```

Comment:
I think the confiedence on set value is definitely lower than create subkey

Reply:
Also these are not the correct function names for C#

Reply:
No they aren't and can't find what language I found those in but there's a lot of them and easy to get a couple mixed though good Issue record so far.  I've corrected.  Thanks.